### PR TITLE
Update shared-channels.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
as per discussion with Jason Lewis

### CODE REVIEW

**Comment by RuiPereiraTabares** [comment]
@JasonLewisMSFT  FYI

**Comment by JasonLewisMSFT** [comment]
Changes look good!

File: Teams/shared-channels.md [hunk 1]
```diff
@@ -61,7 +61,8 @@ Deleted teams can be restored within 30 days after deletion. When a team is dele
 When a team is archived, individual sharing will remain intact, but sharing with teams other than the parent team will be removed. When archived team is unarchived, all shared channels will be restored with individual sharing only.
 
 ## Adding and removing owners and members
-
+>[!NOTE] 
+>Externally Authenticated users such as Guest or Guest users converted to members (in their usertype property) are not supported to be added in a shared channel.
 A shared channel owner can't be removed through the Teams client if they are the last owner of one or more shared channels.
```
Comment (JohanFreelancer9) [inline Teams/shared-channels.md:66]:
```suggestion
> [!NOTE] 
> Externally authenticated users such as guests or guest users converted to members (in their user type property) are not supported to be added to a shared channel.

A shared channel owner can't be removed through the Teams client if they are the last owner of one or more shared channels.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RuiPereiraTabares, kindly commit the suggested changes. Many thanks.

**Comment by RuiPereiraTabares** [comment]
@JohanFreelancer9 done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**